vector/src/main/java/im/vector/app/core/ui/views/ReadReceiptsView.kt

```diff
@@ -60,66 +53,58 @@ class ReadReceiptsView @JvmOverloads constructor(
     }
 
     fun render(readReceipts: List<ReadReceiptData>, avatarRenderer: AvatarRenderer) {
-        if (readReceipts.isNotEmpty()) {
-            isVisible = true
-            for (index in 0 until MAX_RECEIPT_DISPLAYED) {
-                val receiptData = readReceipts.getOrNull(index)
```

Comment:
I deleted this null check because I verified the call chain and `List<ReadReceiptData>` can never have null elements.